backend/src/src-wsimport/src/main/java/org/eclipse/sw360/wsimport/entitytranslation/WsLicenseToSw360LicenseTranslator.java

```diff
@@ -15,29 +15,28 @@
 import org.eclipse.sw360.wsimport.utility.TranslationConstants;
 import org.eclipse.sw360.datahandler.thrift.licenses.License;
 
-import java.util.Arrays;
 import java.util.HashMap;
 
-import static org.eclipse.sw360.wsimport.utility.TranslationConstants.SUSPECTED;
-
 /**
  * @author ksoranko@verifa.io
  */
 public class WsLicenseToSw360LicenseTranslator implements EntityTranslator<WsLicense, org.eclipse.sw360.datahandler.thrift.licenses.License> {
 
     @Override
     public License apply(WsLicense wsLicense) {
-        String license = wsLicense.getName().replaceFirst(SUSPECTED, "").trim();
-        String licenseWithDashes = license.replaceAll("[\\s+/]","-");
+        String licenseName = wsLicense.getName();
+        String licenseShortName = licenseName
+                .replaceAll("[\\s+/]","-");
```

Comment:
Not part of this PR: This would replace `GPL-2.0+` (outdated but still commonly used) with `GPL-2.0-`?